Integrations/MimecastV2/MimecastV2.py

```diff
@@ -509,7 +630,9 @@ def create_policy_request(policy, option):
             'option': option
         }]
     }
-
+    # Policy ID is not None if it is an update policy request
```

Comment:
please elaborate further on the logic here

Reply:
done